lib/nfc/protocols/mf_desfire/mf_desfire_poller_i.c

```diff
@@ -301,24 +303,35 @@ MfDesfireError mf_desfire_poller_read_file_data(
     size_t size,
     MfDesfireFileData* data) {
     furi_assert(instance);
+    size_t cur_offset, read_size, num_read;
+    MfDesfireError error;
```

Comment:
Compiler thinks that we might not get into for loop, where we update error variable. Please, initialize this variable here

Reply:
No problem. I'll look at it this week.